### PR TITLE
feat: add dutch translations to descriptions

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Adding  dutch descriptions for new presences. (75 left)

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Does not apply in this context

</details>

### CODE REVIEW

File: websites/#/Фугапедия/dist/metadata.json [hunk 1]
```diff
@@ -10,13 +10,14 @@
 	],
 	"description": {
 		"en": "Encyclopedia about the League of Cubism and the server of the universe of SC. Read a lot of cool articles and write your own ones, complementing the most toxic encyclopedia.",
-		"ru": "Энциклопедия про Лигу Кубизма и сервера вселенной СП. Читай много крутых статей и пиши свои, дополняя самую токсчисную энциклопедию."
+		"ru": "Энциклопедия про Лигу Кубизма и сервера вселенной СП. Читай много крутых статей и пиши свои, дополняя самую токсчисную энциклопедию.",
+		"nl": "Encyclopedie over de League van Cubism en de server van het universum van SC. Lees veel coole artikelen en schrijf je eigen artikelen, als aanvulling op de meest giftige encyclopedie."
```
Comment (Bas950) [inline websites/#/Фугапедия/dist/metadata.json:14]: Weet niet of "giftige" het beste is... @QkeleQ10

File: websites/#/대성마이맥/dist/metadata.json [hunk 2]
```diff
@@ -11,10 +11,11 @@
...
-		"vi_VN": "Digital Daesung là công ty giáo dục hàng đầu Hàn Quốc, cung cấp nội dung giáo dục chất lượng cao. Chúng tôi cung cấp dịch vụ giáo dục toàn vẹn cho bậc tiểu học, trung học, cao đẳng và đại học, từ học trực tuyến đến trực tiếp, luyện viết luận và tư vấn nộp hồ sơ đại học."
+		"vi_VN": "Digital Daesung là công ty giáo dục hàng đầu Hàn Quốc, cung cấp nội dung giáo dục chất lượng cao. Chúng tôi cung cấp dịch vụ giáo dục toàn vẹn cho bậc tiểu học, trung học, cao đẳng và đại học, từ học trực tuyến đến trực tiếp, luyện viết luận và tư vấn nộp hồ sơ đại học.",
+		"nl": "Digital Daesung is het vooraanstaande Koreaanse onderwijsbedrijf dat onderwijsinhoud van de hoogste kwaliteit levert. Het bedrijf biedt totale onderwijsdiensten aan basisschool-, middelbare school- en universiteitstudenten, van online en offline colleges tot toetsen, essays schrijven en advies voor toelating tot de universiteit."
```
Comment (Bas950) [inline websites/#/대성마이맥/dist/metadata.json:15]: `basisschool-, middelbare school- en universiteitstudenten,`
basisschool en middelbare ben je niet echt student maar scholier.

en `van online en offline colleges tot toetsen,`
colleges is volgens mij niet het beste heirvoor... @QkeleQ10

File: websites/0-9/10fastfingers/dist/metadata.json [hunk 3]
```diff
@@ -8,10 +8,11 @@
 	"description": {
 		"en": "Test & Improve your Typing Speed with our free Typing Games",
 		"es": "Pruebe y mejore su velocidad de escritura con nuestros juegos de mecanografía gratuitos",
-		"vi_VN": "Thử và cải thiện tốc độ đánh máy của bạn với các trò chơi đánh máy miễn phí của chúng tôi."
+		"vi_VN": "Thử và cải thiện tốc độ đánh máy của bạn với các trò chơi đánh máy miễn phí của chúng tôi.",
+		"nl": "Test & Verbeter uw typesnelheid met onze gratis Typespellen"
```
Comment (Bas950) [inline websites/0-9/10fastfingers/dist/metadata.json:12]:
```suggestion
		"nl": "Test en verbeter uw typsnelheid met onze gratis typspellen"
```

File: websites/A/AMC+/dist/metadata.json [hunk 4]
```diff
@@ -7,13 +7,14 @@
 	"service": "AMC+",
 	"description": {
 		"en": "Award-winning shows and fan favorite movies – AMC+ is your destination for genre-defying storytelling. Ad Free and On Demand with new shows and movies added every week.",
-		"vi_VN": "Các chương trình đạt giải và các bộ phim yêu quý của người dùng - AMC+ là điểm đến của bạn nếu bạn muốn những câu truyện bất chấp thể loại. Không quảng các và theo nhu cầu các chương trình và bộ phim mới."
+		"vi_VN": "Các chương trình đạt giải và các bộ phim yêu quý của người dùng - AMC+ là điểm đến của bạn nếu bạn muốn những câu truyện bất chấp thể loại. Không quảng các và theo nhu cầu các chương trình và bộ phim mới.",
+		"nl": "Award winnende shows en fan favoriete films - AMC+ is uw bestemming voor genre-bepalende verhaal vertelling. Zonder advertenties en On Demand met elke week nieuwe shows en films."
```
Comment (Bas950) [inline websites/A/AMC+/dist/metadata.json:11]: `Award winnende shows en fan favoriete films - AMC+ is uw bestemming voor genre-bepalende verhaal vertelling.`
@QkeleQ10 Hier zitten denk ik meerdere fouten in.

`Zonder advertenties en On Demand met elke week nieuwe shows en films.`
Misschien is dit beter:
`Advertentievrij en on-demand met elke week nieuwe shows en films.`

File: websites/A/ARD Mediathek/dist/metadata.json [hunk 5]
```diff
@@ -8,10 +8,11 @@
 	"description": {
 		"de": "Die ARD Mediathek ist das gemeinsame Video-on-Demand-Angebot der Landesrundfunkanstalten und Gemeinschaftseinrichtungen der ARD. Das Angebot wurde wie die ehemals eigenständige Das Erste: Mediathek erstmals 2007 auf der Internationalen Funkausstellung in Berlin vorgestellt und im Mai 2008 gestartet.",
 		"en": "ARD Mediathek is a shared video-on-demand streaming service offered by German public-sector broadcasters of ARD.",
-		"vi_VN": "ARD Mediathek là dịch vụ phát video trực tuyến theo nhu cầu của các đài truyền hình công Tiếng Đức của ARD."
+		"vi_VN": "ARD Mediathek là dịch vụ phát video trực tuyến theo nhu cầu của các đài truyền hình công Tiếng Đức của ARD.",
+		"nl": "ARD Mediathek is een gedeelde video-on-demand streamingdienst die door de Duitse openbare omroep sector van ARD wordt aangeboden."
```
Comment (Bas950) [inline websites/A/ARD Mediathek/dist/metadata.json:12]:
```suggestion
		"nl": "ARD Mediathek is een gedeelde video-on-demand streamingdienst die door de Duitse publieke omroep van ARD wordt aangeboden."
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Denk dat @QkeleQ10 hier beter naar kan kijken lol

File: websites/D/Discord Resources/dist/metadata.json [hunk 6]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "Discord Resources",
 	"description": {
-		"en": "A list of tons of useful Discord resources and utilities for all types of users, from beginners to power users."
+		"en": "A list of tons of useful Discord resources and utilities for all types of users, from beginners to power users.",
+		"nl": "Een lijst van tonnen nuttige Discord bronnen en hulpprogramma's voor alle soorten gebruikers, van beginners tot power users."
```
Comment (QkeleQ10) [inline websites/D/Discord Resources/dist/metadata.json:10]:
```suggestion
		"nl": "Een grote lijst van nuttige Discord-bronnen en -hulpprogramma's voor alle soorten gebruikers: van beginners tot powerusers."
```

File: websites/F/Flightradar24/dist/metadata.json [hunk 7]
```diff
@@ -6,15 +6,16 @@
 	},
 	"service": "Flightradar24",
 	"description": {
-		"en": "The world’s most popular flight tracker. Track planes in real-time on our flight tracker map and get up-to-date flight status & airport information."
+		"en": "The world’s most popular flight tracker. Track planes in real-time on our flight tracker map and get up-to-date flight status & airport information.",
+		"nl": "'S werelds meest populaire vlucht tracker. Volg vliegtuigen in real-time op onze vlucht tracker kaart en krijg up-to-date vluchtstatusen & luchthaveninformatie."
```
Comment (QkeleQ10) [inline websites/F/Flightradar24/dist/metadata.json:10]:
```suggestion
		"nl": "'s Werelds meest populaire vluchttracker. Volg vliegtuigen in real-time op onze vluchtenkaart en krijg up-to-date vluchtstatussen & luchthaveninformatie."
```

File: websites/G/GAMEE/dist/metadata.json [hunk 8]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "GAMEE",
 	"description": {
-		"en": "Play games, get tickets and WIN $$$. More tickets, more chances to WIN!"
+		"en": "Play games, get tickets and WIN $$$. More tickets, more chances to WIN!",
+		"nl": "Speel spelletjes, krijg tickets en WIN $$$. Hoe meer tickets, meer kansen om te WINNEN!\""
```
Comment (QkeleQ10) [inline websites/G/GAMEE/dist/metadata.json:10]:
```suggestion
		"nl": "Speel spelletjes, krijg tickets en WIN $$$. Hoe meer tickets, meer kansen om te WINNEN!"
```

File: websites/G/Genshin Impact Map/dist/metadata.json [hunk 9]
```diff
@@ -6,7 +6,8 @@
 	},
 	"service": "Genshin Impact Map",
 	"description": {
-		"en": "Genshin Impact Interactive Map of all locations locations descriptions guides and more."
+		"en": "Genshin Impact Interactive Map of all locations locations descriptions guides and more.",
+		"nl": "Genshin Impact Interactieve kaart van alle locaties, locaties beschrijvingen gidsen en meer."
```
Comment (QkeleQ10) [inline websites/G/Genshin Impact Map/dist/metadata.json:10]:
```suggestion
		"nl": "Genshin Impact's interactieve kaart met alle locaties, locatiebeschrijvingen, gidsen en meer."
```
This English description is a mess

File: websites/G/Google Play/dist/metadata.json [hunk 10]
```diff
@@ -7,11 +7,12 @@
 	"service": "Google Play",
 	"description": {
 		"en": "Enjoy millions of the latest Android apps, games, music, movies, TV, books, magazines & more.",
-		"bg": "Насладете се на милиони най-нови приложения за Android, игри, музика, филми, телевизия, книги, списания и др."
+		"bg": "Насладете се на милиони най-нови приложения за Android, игри, музика, филми, телевизия, книги, списания и др.",
+		"nl": "Geniet van miljoenen van de nieuwste Android apps, games, muziek, films, TV, boeken, tijdschriften & meer."
```
Comment (QkeleQ10) [inline websites/G/Google Play/dist/metadata.json:11]:
```suggestion
		"nl": "Geniet van miljoenen Android-apps, games, muziek, films, TV, boeken, tijdschriften & meer."
```

File: websites/G/Google's Coding Competitions/dist/metadata.json [hunk 11]
```diff
@@ -7,10 +7,11 @@
 	"service": "Google's Coding Competitions",
 	"description": {
 		"en": "Google's Coding Competitions (Code Jam, Hash Code, and Kick Start) enthrall, challenge, and test coders around the world. Try your hand at one, or all three.",
-		"vi_VN": "Google's Coding Competitions (Code Jam, Hash Code, and Kick Start) cuốn hút, thử thách lập trình viên khắp thế giới. Hãy thử một, hoặc có thể cả ba."
+		"vi_VN": "Google's Coding Competitions (Code Jam, Hash Code, and Kick Start) cuốn hút, thử thách lập trình viên khắp thế giới. Hãy thử một, hoặc có thể cả ba.",
+		"nl": "Google's codeerwedstrijden (Code Jam, Hash Code en Kick Start) boeien, dagen uit en testen programmeurs van over de hele wereld. Probeer er een te spelen, of alle drie."
```
Comment (QkeleQ10) [inline websites/G/Google's Coding Competitions/dist/metadata.json:11]:
```suggestion
		"nl": "Google's codeerwedstrijden (Code Jam, Hash Code en Kick Start) boeien en testen programmeurs van over de hele wereld en dagen ze uit. Probeer er een te spelen, of alle drie."
```

**Review by QkeleQ10 — CHANGES_REQUESTED** [review]
Didn't correct every single thing, as it just was way too much to look over. Looks fine

wat een kutklus

**Comment by DarkVillager** [comment]
> Didn't correct every single thing, as it just was way too much.
> 
> wat een kutklus

Je mag ooi deze pr overnemen

File: websites/A/Animekimi/dist/metadata.json [hunk 12]
```diff
@@ -6,12 +6,11 @@
 	},
 	"service": "Animekimi",
 	"description": {
-		"en": "Watch anime online cartoons Thai dubbed anime Thai subtitled anime latest episodes.",
-		"th": "ดูอนิเมะออนไลน์ การ์ตูน อนิเมะพากย์ไทย อนิเมะซับไทย ตอนใหม่ล่าสุด",
-		"vi_VN": "Xem những anime mới nhất trực tuyến có phụ đề và thuyết minh Tiếng Thái."
```
Comment (DarkVillager) [inline websites/A/Animekimi/dist/metadata.json:11]: Not sure if this is accurate anymore due to changing the English one.

File: websites/K/Kayoanime/dist/metadata.json [hunk 13]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "Kayoanime",
 	"description": {
-		"en": "Kayoanime is a website where you can download anime in dual audio 1080p through Google drive."
+		"en": "Kayoanime is a website where you can download anime in 1080p with dual audio through Google drive.",
```
Comment (DarkVillager) [inline websites/K/Kayoanime/dist/metadata.json:9]: Grammar sounded off

**Review by QkeleQ10 — APPROVED** [review]

File: websites/N/namu.wiki/dist/metadata.json [hunk 14]
```diff
@@ -6,11 +6,12 @@
 	},
 	"service": "namu.wiki",
 	"description": {
-		"en": "\"namuwiki, The tree of knowledge that you nurtures.\" namuwiki(나무위키) is a Korean language wiki based on \"the seed\" engine.",
-		"ko_KR": "\"나무위키, 여러분이 가꾸어 나가는 지식의 나무.\" 나무위키(namuwiki)는 \"the seed\" 엔진 기반의 한국어 위키이다."
+		"en": "\"namuwiki, The tree of knowledge that nurtures you.\" namuwiki(나무위키) is a Korean language wiki based on \"the seed\" engine.",
```
Comment (DarkVillager) [inline websites/N/namu.wiki/dist/metadata.json:9]: typo fixed

File: websites/N/Notion/dist/metadata.json [hunk 15]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "Notion",
 	"description": {
-		"en": "Notion is a notetaking software and project management software that is used for note-taking, task management, project management, knowledge management, and personal knowledge management. The app uses databases and markdown pages for use in personal and collaboration work."
+		"en": "Notion software are used for note-taking, task management, project management, knowledge management, and personal knowledge management. The app uses databases and markdown pages for use in personal and collaboration work.",
```
Comment (DarkVillager) [inline websites/N/Notion/dist/metadata.json:9]: Old one seemed weird

File: websites/P/padlet/dist/metadata.json [hunk 16]
```diff
@@ -6,10 +6,11 @@
 	},
 	"service": "padlet",
 	"description": {
-		"en": "From your hobby to your career, your class notes to your final exam, your mood board to your runway show, padlets help you organize your life."
+		"en": "From your hobby to your career, your class notes to your final exam, your mood board to your runway show, padlets help you organize your life.",
+		"nl": "Van je hobby tot je carrière, van je aantekeningen tot je eindexamen, van je stemmingsbord tot je catwalkshow, padlets helpen je je leven te organiseren."
```
Comment (DarkVillager) [inline websites/P/padlet/dist/metadata.json:10]: @QkeleQ10

File: websites/P/paimon.moe/dist/metadata.json [hunk 17]
```diff
@@ -6,16 +6,17 @@
 	},
 	"service": "paimon.moe",
 	"description": {
-		"en": "Your best Genshin Impact companion! Help you plan what to farm with ascension calculator, also track your progress with todo and wish counter.",
+		"en": "Your best Genshin Impact companion! Help you plan what to farm with ascension calculator, also track your progress with a todo and wish counter.",
```
Comment (DarkVillager) [inline websites/P/paimon.moe/dist/metadata.json:9]: Sounded off without the a

**Comment by EncryptedDev** [comment]
Any update here or should I close the PR until you're done?

**Comment by DarkVillager** [comment]
> Any update here or should I close the PR until you're done?

I’ll be pushing a commit today

**Comment by DarkVillager** [comment]
I will come back to this when I can.

**Pull request closed without merging**